proto/rill/runtime/v1/queries.proto

```diff
@@ -274,6 +280,14 @@ message MetricsViewComparisonValue {
   google.protobuf.Value delta_rel = 5;
 }
 
+message TimeZoneAdjustment {
+  oneof adjustment {
+    int32 offset = 1;
+    google.protobuf.Timestamp origin = 2;
+    string time_zone = 3;
+  }
+}
```

Comment:
It looks like the implementation does not handle `offset` and `origin`? In that case, let's just pass `string time_zone` directly

Reply:
I thought it could be used in the future. I can remove it, of course.